docs/reference_guides/model_libraries/generic/unit_models/heat_exchanger_lc.rst

```diff
@@ -0,0 +1,227 @@
+Heat Exchanger (Lumped Capacitance)
+===================================

```

Comment:
Feel free to add a line here taking credit for the contribution. Not only do we want to acknowledge external input, but it might also encourage others to contribute as well.